tests/bson/bson-document-array-access-001.phpt

```diff
@@ -0,0 +1,39 @@
+--TEST--
+MongoDB\BSON\Document array access
+--FILE--
+<?php
+
+require_once __DIR__ . '/../utils/basic.inc';
+
+$document = MongoDB\BSON\Document::fromPHP([
+    'foo' => 'bar',
+    'bar' => 'baz',
+    'int64' => new MongoDB\BSON\Int64(123),
+]);
+
+var_dump(isset($document['foo']));
+var_dump(isset($document['int64']));
+var_dump(isset($document['baz']));
+
+var_dump($document['foo']);
+var_dump($document['int64']);
+
+echo throws(function() use ($document) {
+    var_dump($document['baz']);
+}, MongoDB\Driver\Exception\RuntimeException::class), "\n";
```

Comment:
Should this be reserved for an error test? I suppose you can use a new file since it'd warrant a unique description.